docs/general/events.md

```diff
@@ -3,13 +3,34 @@ linkTitle: Events
 aliases: [docs/specs/semconv/general/events-general]
 --->
 
-# Semantic Conventions for Event Attributes
+# Semantic Conventions for Events
 
 **Status**: [Experimental][DocumentStatus]
 
-This document describes the attributes of standalone Events that are represented
+This document describes the characteristics of standalone Events that are represented
 in the data model by `LogRecord`s.
 
+Semantically, an Event is a named occurrence at an instant in time. It signals that
+"this thing happened at this time" and provides additional specifics about the occurrence.
+Examples of Events might include things like uncaught exceptions, button clicks, user logout,
+network connection severed, etc.
+
+In OpenTelemetry, Events are implemented as a specific type of `LogRecord` that conforms to
+the conventions included here, and Events
+[have their own API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md).
+The API abstracts away knowledge of `LogRecord` so that users only deal with Event
+semantics.
+
+In addition to a required name, an Event may contain a _payload_, which can be either structured or unstructured.
+In its implementation, the Event _payload_ will constitute the `Body` of the `LogRecord`.
+Like all other OpenTelemetry signals, an Event has optional attribute metadata that helps describe
+the event context.
+
+Over time, some Events will be specified by OpenTelemetry and will have documented payload structure,
+field semantics, and stability and requirement levels. Other events may be user-defined and carry
+bespoke user semantics. The name of the Event determines if an established payload structure is applicable
+or not.
+
```

Comment:
Can you clarify what you mean in the last sentence? By established payload structure, do you mean schema of the event? I am of the understanding that any event names documented in this repo will also have its payload structure documented, so it's not clear what you mean by a schema not being application for certain event names.

Reply:
> By established payload structure, do you mean schema of the event?

Yeah, a schema is what would be used to describe the structure in most cases. If you think that using the word schema here helps, we can try it out. I mostly didn't want to get too specific about a thing that doesn't exist yet.

> I am of the understanding that any event names documented in this repo will also have its payload structure documented, so it's not clear what you mean by a schema not being applicable for certain event names.

The intent, without getting too deep into the event registry and schemas (both of which are germinating), was to suggest that the name of the event determines if a predefined structure applies. That is, an event like `otel.button.click` should be in the registry within the spec and have a defined structure (schema), but `jimmy.door.knock` might not.

Reply:
I am still unclear, if you are suggesting a convention for event names for events in the semconv registry, like `otel` prefix or just that if an event name is defined in the registry it also has its payload structure defined. If it's the latter, I suggest rewording to something like this - "All events defined in the semantic conventions include both the event name and its payload structure"..

This brings up a question whether we allow defining event names  in semantic conventions for external events, like Kubernetes events, for which we cannot defining the structure since it's not in our control.

Reply:
> if you are suggesting a convention for event names for events in the semconv registry

No, this would not be the right PR for that.

> if an event name is defined in the registry it also has its payload structure defined

Basically. The intent is more like "If you get an event, how do you know if there's a well known structure for the event payload? You look at the name. The body itself doesn't tell if there is a schema, the name is what determines it."

But I think I like your idea `"All events defined in the semantic conventions include both the event name and its payload structure"`. Let me try something more like that.

Reply:
Ok, try out that new wording.